src/main/groovy/org/transmartproject/batch/highdim/proteomics/data/ProteomicsDataWriter.groovy

```diff
@@ -0,0 +1,52 @@
+package org.transmartproject.batch.highdim.proteomics.data
+
+import org.springframework.batch.core.configuration.annotation.JobScope
+import org.springframework.batch.item.ItemWriter
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.beans.factory.annotation.Value
+import org.springframework.jdbc.core.JdbcTemplate
+import org.springframework.jdbc.core.simple.SimpleJdbcInsert
+import org.springframework.stereotype.Component
+import org.transmartproject.batch.clinical.db.objects.Tables
+import org.transmartproject.batch.db.DatabaseUtil
+import org.transmartproject.batch.highdim.datastd.StandardDataValue
+
+import static org.transmartproject.batch.clinical.db.objects.Tables.schemaName
+import static org.transmartproject.batch.clinical.db.objects.Tables.tableName
+
+/**
+ * Writes the proteomics data to de_subject_protein_data.
+ */
+@Component
+@JobScope
+class ProteomicsDataWriter implements ItemWriter<StandardDataValue> {
```

Comment:
It actually needs `TripleStandardDataValue` items (to pass to `ProteomicsDataRowConverter::convertProteomicsDataValue`).